sdk/standbypool/arm-standbypool/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
     
-## 1.0.0-beta.1 (2024-04-10)
+## 1.0.0 (2024-09-11)
 
 ### Features Added
 
-The package of @azure/arm-standbypool is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart ).
+The package of @azure/arm-standbypool is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart).
```

Comment:
Need to confirm the changelog here.

Reply:
no, the GA changelog is from hlc

Reply:
for preview tsp packages the log is ```Initial release of the Azure MongoCluster package```
need confirm the log with tsp ga packages.